packages/block-editor/src/components/rich-text/index.js

```diff
@@ -115,6 +115,15 @@ function RichTextWrapper(
 	},
 	forwardedRef
 ) {
+	if ( multiline ) {
+		deprecated( 'wp.blockEditor.RichText multiline prop', {
+			since: '6.1',
+			version: '6.2',
```

Comment:
I think 6.2 is way too close. Even though everything suggests that the multiline option was minimally used in the ecosystem, it has been part of a stable API for a long time.

For comparison, take the migration of classnames in 5.2 following the introduction of the block editor module. The classes were never even a public API, but we still gave it a year (which was very generous, IMO).

So I think 6.3 should be the absolute minimum.

Reply:
Alright :) I think it depends in how much it's communicated, but I agree that a bit longer is probably best.